CRM/Core/BAO/PrevNextCache.php

```diff
@@ -200,8 +206,12 @@ public static function markConflict($id1, $id2, $cacheKey, $conflicts) {
         $pncUp = new CRM_Core_DAO_PrevNextCache();
         $pncUp->id = $pncFind->id;
         if ($pncUp->find(TRUE)) {
-          $pncUp->data     = serialize($data);
+          $expiry = (float) round(microtime(1)) + (float) self::DEFAULT_TTL;
+          $date = new DateTime();
+          $date->setTimeStamp($expiry);
+          $pncUp->data = serialize($data);
           $pncUp->cachekey = "{$cacheKey}_conflicts";
+          $pncUp->expiry_date = CRM_Utils_Date::convertDateToLocalTime($date, 'Y-m-d H:i:s');
```

Comment:
@mattwire is right - it doesn't make sense to convert dates when writing them to a timestamp field.

Reply:
Also - this is really grey. There are 2 uses for prev_next_cache - dedupe & search. I understood that they were currently treated somewhat differently - ie. only search uses the PrevNext interface. I think the dedupe caches persist for quite a long time - it feels like another can of worms

Reply:
Does Dedupe call Civi::cache('prevnext') at all in SQL mode? I feel like dedupe should expire like Search does to me but agree that might be a meaninful change but would like @totten to discuss